### ISSUE TITLE
Any thoughts into adding more than one author?

### ISSUE THREAD

**Issue by varelycode** [git-duet/git-duet#96]
Often times our team finds ourselves trio-ing or mobbing. Would be nice if there was a git mob or git party option that allows more than two authors initials to be added as contribution.

**Comment by jszwedko** [comment]
Yeah, that'd be a good idea. We do have the option to rotate the committers for "mob programming", but we could add all as `Co-Authored-By`.

**Comment by kynetiv** [comment]
> ...but we could add all as Co-Authored-By.

Mobbing support seems partially working already. I've noticed that it will add the additional `Co-Authored-By` lines on a commit when I have `export GIT_DUET_CO_AUTHORED_BY=1` set. 

The only problem I've seen though is that a `git commit --amend`, will re-apply all the `Co-Authored-By` lines again, stacking them on top of any existing `Co-Authored-By` lines.

**Comment by lx4r** [comment]
For me the mobbing support isn't working.

My config:

```fish
❯ echo $GIT_DUET_ALLOW_MULTIPLE_COMMITTERS
1
❯ echo $GIT_DUET_CO_AUTHORED_BY
```

`.git-authors`:

```
authors:
  b: Bob
  a: Alice
  c: Carol
email:
  domain: example.com
```

Executed commands:

```fish
❯ git duet a b c
GIT_AUTHOR_NAME='Alice'
GIT_AUTHOR_EMAIL='alice@example.com'
GIT_COMMITTER_NAME='Bob'
GIT_COMMITTER_EMAIL='bob@example.com'
❯ git duet-commit -v
```

The commit message I get:

```
Signed-off-by: Bob <bob@example.com>
```

What can I do to fix this?

Thanks in advance for the help :)

**Comment by halfghaninne** [comment]
Could someone clarify for me where this `GIT_DUET_CO_AUTHORED_BY` gets set? Is this in my .gitconfig?

**Comment by jszwedko** [comment]
> Could someone clarify for me where this `GIT_DUET_CO_AUTHORED_BY` gets set? Is this in my .gitconfig?

It's an environment variable that you would set in your shell.
